packages/api/src/routes/medical/patient.ts

```diff
@@ -271,7 +274,7 @@ router.get(
  *
  * @param req.cxId The customer ID.
  * @param req.param.id The ID of the patient whose data is to be returned.
- * @return status of querying for the Patient's consolidated data.
+ * @return array of statuses for querying the Patient's consolidated data.
```

Comment:
This will break the API for current customers. I'd try to keep it backwards compatible. We can chat more on Slack if needed.

Reply:
Good point - to not introduce a breaking change immediately we have 2 options it seems

- Return status like we did before and add queries
- Create another endpoint with the new resp

I like the 2 option better lmk what you think

Reply:
Since we're currently returning a JSON object, I think adding to it could work:
- `status` is the latest one;
- `queries` contain all the status.

Then we can mark `status` as deprecated and update the docs accordingly. We can communicate that to cxs and give them lots of time to update it to only rely on `queries`.

I like that because it doesn't break the behavior and cxs don't need to update their code to work with a new endpoint.

Thoughts?